tasks/prelim.yml

```diff
@@ -92,6 +92,14 @@
       - audit
       - RHEL-07-010240
 
+- name: "PRELIM | Find all sudoers files."
+  command: "find /etc/sudoers /etc/sudoers.d/ -type f"
```

Comment:
we should exclude files that end in '~' and files that contain '.' since sudo ignores those.